circle.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   linux:
     docker:
-      - image: swift:4.1
+      - image: swift:5.1
```

Comment:
The CI error should be fixable by using `vapor/swift:5.1` which includes packages Vapor needs like OpenSSL

Reply:
Ah, of course. Thanks!